specs/merge/fork-choice.md

```diff
@@ -24,7 +29,44 @@ This is the modification of the fork choice according to the executable beacon c
 
 *Note*: It introduces the process of transition from the last PoW block to the first PoS block.
 
-### Helpers
+## Protocols
+
+### `ExecutionEngine`
+
+The following methods are added to the `ExecutionEngine` protocol for use in the fork choice:
+
+#### `set_head`
+
+Re-organizes the execution payload chain and corresponding state to make `block_hash` the head.
+
+The body of this function is implementation dependent.
+The Consensus API may be used to implement this with an external execution engine.
+
+```python
+def set_head(self: ExecutionEngine, block_hash: Hash32) -> bool:
```

Comment:
This function is somewhat out of the spec and will unlikely become a part of it in the future. Do we really want to keep it here?

Reply:
It's not called in the eth2 specs yet, I just added it here for completeness. If we're not going to have it, we can leave it out. I still like the idea of updating the head of an external execution-client though, it's important to detect reorgs without the chain being live (i.e. no new block insertions to keep the head updated).